eng/Versions.props

```diff
@@ -9,27 +9,34 @@
     <UsingToolSymbolUploader>true</UsingToolSymbolUploader>
     <UsingToolVSSDK>true</UsingToolVSSDK>
   </PropertyGroup>
+
   <!-- Version number computation -->
   <PropertyGroup>
     <PreReleaseVersionLabel>beta</PreReleaseVersionLabel>
-    <FSCoreMajorVersion>4.6</FSCoreMajorVersion>
+    <FSLanguageVersion>4.6</FSLanguageVersion>
+    <FSCoreMajorVersion>$(FSLanguageVersion)</FSCoreMajorVersion>
     <FSCorePackageVersion>$(FSCoreMajorVersion).3</FSCorePackageVersion>
     <FSCoreVersionPrefix>$(FSCoreMajorVersion).0</FSCoreVersionPrefix>
     <FSCoreVersion>$(FSCoreVersionPrefix).0</FSCoreVersion>
+    <FSharpCoreShippedPackageVersion>4.6.2</FSharpCoreShippedPackageVersion>                                <!-- The current published nuget package -->
```

Comment:
The arcade update bot will also screw up the formatting of these.  To keep them relevant add them on the line before.